wbooks/src/app/screens/booklist/index.js

```diff
@@ -1,24 +1,17 @@
 import React, { Component } from 'react';
 import { View, StyleSheet } from 'react-native';
 import BookItem from './components/BookItem';
+import styles from './styles';
 
 class BookList extends Component {
 
 	render() {
 		return (
 			<View style={styles.container}>
-                <BookItem />
+        <BookItem />
 			</View>
 		);
 	}
 }
 
-const styles = StyleSheet.create({
-	container: {
-		flex: 1,
-        alignItems: 'center',
-        padding:20,
-	}
-});
-
 export default BookList;
```

Comment:
always add an enter at the end of each file to avoid this

Reply:
done!